### PR TITLE
fix zz links

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #3682 

From what I saw, MarkDig migration doesn't like these zz links much, so fixing a few of them now. Will try more tomorrow.

### CODE REVIEW

File: docs/framework/wcf/feature-details/wcf-discovery-object-model.md [hunk 1]
```diff
@@ -74,14 +74,14 @@ WCF Discovery consists of a set of types that provide a unified programming mode
...
   
 ## DiscoveryEndpoint  
- <xref:System.ServiceModel.Discovery.DiscoveryEndpoint> represents a standard endpoint with a fixed discovery contract. It is used by a service or client to send or receive discovery messages. By default, <xref:System.ServiceModel.Discovery.DiscoveryEndpoint> is set to use <!--zz <xref:System.ServiceModel.Discovery.DiscoveryMode.Managed>--> `Managed` mode and the WSDiscovery11 WS-Discovery version.  
+ <xref:System.ServiceModel.Discovery.DiscoveryEndpoint> represents a standard endpoint with a fixed discovery contract. It is used by a service or client to send or receive discovery messages. By default, <xref:System.ServiceModel.Discovery.DiscoveryEndpoint> is set to use <xref:System.ServiceModel.Discovery.ServiceDiscoveryMode.Managed> mode and the WSDiscovery11 WS-Discovery version.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/wcf-discovery-object-model.md]: nit; ServiceDiscoveryMode.Managed is best ?displayProperty=nameWithType

File: docs/framework/wcf/feature-details/wcf-discovery-object-model.md [hunk 2]
```diff
@@ -126,4 +126,4 @@ WCF Discovery consists of a set of types that provide a unified programming mode
  The <xref:System.ServiceModel.Discovery.UdpAnnouncementEndpoint> class is a standard announcement endpoint that is pre-configured for announcement over a UDP multicast binding. By default, <xref:System.ServiceModel.Discovery.UdpAnnouncementEndpoint> is set to use the WSApril2005 WS_Discovery version.  
   
 ## UdpDiscoveryEndpoint  
- The <xref:System.ServiceModel.Discovery.UdpDiscoveryEndpoint> class is a standard discovery endpoint that is pre-configured for discovery over a UDP multicast binding. By default, <xref:System.ServiceModel.Discovery.DiscoveryEndpoint> is set to use the WSDiscovery11 WS-Discovery version and <!--zz <xref:System.ServiceModel.Discovery.UdpDiscoveryEndpoint.Adhoc>--> `Adhoc` mode.
+ The <xref:System.ServiceModel.Discovery.UdpDiscoveryEndpoint> class is a standard discovery endpoint that is pre-configured for discovery over a UDP multicast binding. By default, <xref:System.ServiceModel.Discovery.DiscoveryEndpoint> is set to use the WSDiscovery11 WS-Discovery version and <xref:System.ServiceModel.Discovery.ServiceDiscoveryMode.Adhoc> mode.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/wcf-discovery-object-model.md]: nit: also best Adhoc?displayProperty=nameWithType

**Review by rpetrusha — APPROVED** [review]
This looks really good, @mairaw. Left a few nits, and also what I think are the correct xrefs for two remaining occurrences of "zz" in these articles.

File: docs/framework/wcf/feature-details/accessing-wcf-services-with-a-windows-store-client-app.md [hunk 3]
```diff
@@ -103,11 +103,11 @@ public partial class Service1Client : System.ServiceModel.ClientBase<MetroWcfCli
...
-1.  <xref:System.ServiceModel.SecurityMode.None>  
+1. <xref:System.ServiceModel.SecurityMode.None>  
   
-2.  <xref:System.ServiceModel.SecurityMode.Transport>  
+2. <xref:System.ServiceModel.SecurityMode.Transport>  
   
-3.  <!--zz <xref:System.ServiceModel.SecurityMode.TransportWithMessageCredentials> --> `System.ServiceModel.SecurityMode.TransportWithMessageCredentials`
+3. <xref:System.ServiceModel.SecurityMode.TransportWithMessageCredential>
   
 4.  <!--zz <xref:System.ServiceModel.SecurityMode.TransportCredentialOnly>  --> `System.ServiceModel.SecurityMode.TransportCredentialOnly`
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/framework/wcf/feature-details/accessing-wcf-services-with-a-windows-store-client-app.md]: I've changed to Message which was the only other enum value left at https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.securitymode.

**Pull request merged**